### PR TITLE
Code Style Change - Method Parameters on new line from parentheses

### PR DESCRIPTION
**Pull request by ericdecanini**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [x] Technical
- [ ] Other :

## Content

Applies the codes style change to move (multiline) method parameters on a new line from both the opening and closing parentheses

## Motivation and context

Closes https://github.com/vector-im/element-android/issues/5771

See also https://github.com/vector-im/element-android/pull/5644

I tried to isolate this change to the mentioned issue as much as possible, but it seems the autoformatter wanted to correct the inconsistencies that weren't on the current code style

I tested conflicts for this by merging this branch into the [Tchap fork](https://github.com/tchapgouv/tchap-android) and this seemed to introduce only a few more easy-to-resolve conflicts than there would be when merging with develop

I figured if the forks perform the same code style change strategy as undertaken to make the changes on this PR, they can encounter even less conflicts. I will document this strategy later, but in a nutshell, that strategy is:

1. Strip down the `.editorconfig` file to only the changes being made

```
[{*.gradle.kts,*.kt,*.kts,*.main.kts}]
ij_kotlin_method_parameters_new_line_after_left_paren = true
ij_kotlin_method_parameters_right_paren_on_new_line = true
```

2. Reformat the modules with file mask `.kt`

## Screenshots / GIFs

N/A

## Tests

- Checkout Tchap fork
- Add the original element repo as upstream and fetch all the branches
- Merge this branch with Tchap develop

You will see that there are a lot of conflicts, but most of these conflicts come from changes that are on our repo's develop and not the changes on this branch specifically. If you repeat the steps above but instead merge upstream/develop into origin/develop, you will see that most of those conflicts are still present and this branch adds only few.

## Tested devices

- [ ] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [ ] Changes has been tested on an Android device or Android emulator with API 21
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [x] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

### CODE REVIEW

File: library/jsonviewer/src/main/java/org/billcarsonfr/jsonviewer/JSonViewerDialog.kt [hunk 1]
```diff
@@ -27,9 +27,9 @@ import com.airbnb.mvrx.Mavericks
 class JSonViewerDialog : DialogFragment() {
 
     override fun onCreateView(
-        inflater: LayoutInflater,
-        container: ViewGroup?,
-        savedInstanceState: Bundle?
+            inflater: LayoutInflater,
```
Comment (ouchadam) [inline library/jsonviewer/src/main/java/org/billcarsonfr/jsonviewer/JSonViewerDialog.kt:30]: it looks like these formatting changes could be done separately from the parameter changes (would also help reduce the noise is we're aiming to incrementally change the formatting rules)

Comment (ericdecanini) [reply library/jsonviewer/src/main/java/org/billcarsonfr/jsonviewer/JSonViewerDialog.kt:30]: I tried isolating this change as much as possible, but I found that we can't do "I only want to reformat this change". Even if you removed all other lines from editorconfig, the IDE wants to follow each rule as either true or false.

In this case, this is a case of us having continuation indents enabled. If we disabled it, the rest of the project that is using continuation indents will change.

This is a consequence of us having inconsistent formatting throughout the project

Comment (ouchadam) [reply library/jsonviewer/src/main/java/org/billcarsonfr/jsonviewer/JSonViewerDialog.kt:30]: my bad I meant that we could format the codebase without the editconfig changes in this file to get a baseline, then all the PRs afterwards will only contain the new changes

Comment (ericdecanini) [reply library/jsonviewer/src/main/java/org/billcarsonfr/jsonviewer/JSonViewerDialog.kt:30]: That approach could work too, though I guess the PR for that wouldn't be this one. I'll go ahead and make a new one

Comment (ericdecanini) [reply library/jsonviewer/src/main/java/org/billcarsonfr/jsonviewer/JSonViewerDialog.kt:30]: Closing this for now. Will reapproach after merging https://github.com/vector-im/element-android/pull/5805

**Pull request closed without merging**